packages/e2e-tests/cypress/integration/eligibility/householder-planning/lpd-letter-decision-date/lpd-letter-decision-date-steps.js

```diff
@@ -0,0 +1,122 @@
+import {allowedDatePart, getFutureDate, getPastDate} from "../../../../support/common/getDate";
+import {getDate, getMonth, getYear} from "date-fns";
+import {
+  enterDateDecisionReceived
+} from "../../../../support/eligibility/date-decision-received/enter-date-decision-received";
+import {getDateHouseholderDecisionReceivedDay} from "../../../../support/eligibility/page-objects/date-decision-received-po";
+import {clickContinueButton} from "../../../../support/common/clickContinueButton";
+import {verifyErrorMessage} from "../../../../support/common/verify-error-message";
+import {getBackLink, getErrorMessageSummary} from "../../../../support/common-page-objects/common-po";
+import {
+  verifyHouseholderDecisionDatesHighlights
+} from "../../../../support/eligibility/date-decision-received/enter-date-householder-decision-received";
+import {getPlanningApplicationHouseholderDecisionError} from "../../../../support/eligibility/page-objects/date-decision-received-po";
+import {
+  selectListedBuildingDecision
+} from '../../../../support/eligibility/listed-building/select-listed-building-decision';
+import {
+  enterDateHouseholderDecisionReceived
+} from '../../../../support/eligibility/date-decision-received/enter-date-householder-decision-received';
+const pageHeading = 'What\'s the decision date on the letter from the local planning department?';
+const pageTitle = 'What\'s the decision date on the letter from the local planning department? - Before you start - Appeal a householder planning decision - GOV.UK';
+const url = `/decision-date-householder`;
+const typeOfPlanningPageUrl = `before-you-start/type-of-planning-application`;
+const enforcementNoticePageUrl = '/enforcement-notice';
+const grantedOrRefusedPageUrl = '/granted-or-refused-householder';
+const shutterPageUrl = '/you-cannot-appeal';
+
+Given('appellant navigates to decision date received page for householder appeal',()=>{
+  goToAppealsPage(typeOfPlanningPageUrl);
+  selectPlanningApplicationType('Householder');
+  verifyPage(typeOfPlanningPageUrl);
+  clickContinueButton();
+  selectListedBuildingDecision('No');
+  clickContinueButton();
+});
+
+Given('appellant selects the {string}',(application_decision)=>{
+  selectPlanningApplicationDecision(application_decision);
+  clickContinueButton();
+});
+
+Given('appellant is on the what date was the decision received page',()=>{
+  verifyPage(url);
+  verifyPageTitle(pageTitle);
+  verifyPageHeading(pageHeading);
+});
+
+When('appellant enters the date lesser than the deadline date for {string}',(application_decision)=>{
+  if(application_decision==='Granted'){
+    const validDate = getPastDate(allowedDatePart.MONTH, 3);
+    enterDateHouseholderDecisionReceived( {day: getDate(validDate), month: getMonth(validDate) + 1, year: getYear(validDate) } );
+  }else if(application_decision==='Refused'){
+    const validDate = getPastDate(allowedDatePart.MONTH, 2);
```

Comment:
SUGGESTION:- Its appropriate to use WEEKS date part rather than MONTH date part

Reply:
updated